database/migrations/2019_10_06_212436_create_mac_addresses_table.php

```diff
@@ -19,7 +18,7 @@ public function up()
             $table->unsignedBigInteger('user_id');
             $table->string('mac_address', 17);
             $table->text('comment');
-            $table->enum('state', MacAddress::STATES)->default(MacAddress::REQUESTED);
+            $table->enum('state', ['APPROVED','REJECTED','REQUESTED'])->default('REQUESTED');
```

Comment:
I would go as far as saying that we shouldn't even use enums in the database at all. I know it's more safe it SQL also checks if they are enums, but the added value is questionable and it needs altering every time a new option is added to the enum.

Reply:
Agreed. Here, we might change it later. 
Also Laravel now can cast DB values to PHP Enums which I believe triggers a similar validation.

Reply:
I would create a new, seperate migrations for these in #670 or in a follow-up for it

Reply:
The reason for creating a seperate migration is that I would not like to manually change it on the live system. But before that I will check if the actual live schema is different from what migrations produce for us.

Reply:
Added to #671